### ISSUE TITLE
Imported some notes then failed to load the rest, each one failing individually

### ISSUE THREAD

**Issue by camroe** [stefankueng/EvImSync#32]
Import failed on a note about 1/2 way through then all notes after that failed as well. Please find below the first failed note from the temp directory. (I could not attach an .xml file so changed to TXT and attached that)

<note>
    <title>Spicy and Crunchy Garlic Tofu (Kkanpoong Tofu)</title>
    <created>20210201T212137Z</created>
    <updated>20230613T153844Z</updated>
    <note-attributes>
      <author>Cameron Roe</author>
      <latitude>42.34881591796875</latitude>
      <longitude>-82.97162632757721</longitude>
      <altitude>176.07179260253906</altitude>
      <source>desktop.mac</source>
    </note-attributes>
    <content>
      <![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE en-note SYSTEM "http://xml.evernote.com/pub/enml2.dtd"><en-note><ul><li><div>1 16 ounce block extra firm tofu <u>pressed</u> and chopped into 1/2 inch bite sized chunks</div></li><li><div>1 teaspoon salt</div></li><li><div>2 Tablespoons potato starch plus 1 extra teaspoon</div></li></ul><hr/><ul><li><div>7 cloves garlic minced</div></li><li><div>1/4 red onion diced</div></li><li><div>2 Korean green chilis, sliced can sub seeded jalapenos</div></li><li><div>2 whole scallions white parts chopped, green parts sliced on a bias</div></li></ul><hr/><ul><li><div>1 Tablespoon gochagaru (or Siracha)</div></li><li><div>1-1 1/2 Tablespoon soy sauce</div></li><li><div>1 Tablespoon white wine vinegar</div></li><li><div>2 Tablespoons maple syrup</div></li><li><div>1/2 Tablespoon mirin can leave out if you don't have any</div></li><li><div>1 tsp potato starch (this is the extra teaspoon from above)</div></li></ul><hr/><ul><li><div>4+1 Tablespoons vegetable oil</div></li><li><div>1/8 cup dried red chilis can leave out if you don't have any</div></li><li><div>1/2 Tablespoon sesame oil</div></li><li><div>1 Tablespoon toasted sesame seeds</div></li></ul><div>Directions</div><ol><li><div>To a large bowl, add a pinch of salt and 2 tablespoons of potato starch to tofu chunks. Gently toss the tofu to make sure they are all evenly coated. Set aside.</div></li><li><div>Prep your vegetables by mincing the garlic, dicing the onion, slicing the Korean chilis, and chopping up the scallions.</div></li><li><div>Prepare the sauce by mixing together gochugaru, soy sauce, white wine vinegar, maple syrup, mirin, and 1 teaspoon of potato starch. Set aside.</div></li><li><div>To a very large non-stick pan, add 4 tablespoons of vegetable oil over high heat. When oil is very hot, add coated tofu chunks in one layer,making sure they are not touching each other. If they touch, they will stick to one another.</div></li><li><div>(In Batches) Cook the tofu and flip them one by one, until they are brown on all sides (approximately 7 minutes). <u>Remove the tofu chunks from the pan and set them aside on a cooling rack to drain the excess oil</u>. Repeat with the remaining tofu.</div></li><li><div>When all of the tofu has been fried, in the same pan, add 1 tablespoon of oil. Then add the garlic, onions, chilis, and scallions and saute the vegetables until the garlic starts to brown.</div></li><li><div>Reduce the heat to medium-high and add the sauce, stirring it with a wooden spoon until it reduces down into a thick sauce (approximately 30 seconds). Turn off the heat.</div></li><li><div>Gently add back the fried tofu and stir everything together, so that the tofu chunks are evenly coated in your sauce.</div></li><li><div>Garnish the tofu with ½ tablespoon of sesame oil and 1 tablespoon of toasted sesame seeds. Serve immediately.</div></li></ol><div><br/></div></en-note>      ]]>
    </content>
  </note>
[note-3fbb4b1b-4e3b-46ae-a6c6-20549e857a07.txt](https://github.com/stefankueng/EvImSync/files/12313663/note-3fbb4b1b-4e3b-46ae-a6c6-20549e857a07.txt)

**Comment by DMcKenz1e** [comment]
Try turning off (quitting) Onedrive and also in OneNote options turn off sync during the imports. This helped me. DM

**Comment by dhweber** [comment]
Same issue here. I tried quitting both OneDrive and OneNote as well, but it didn't seem to make a difference. Interestingly, after attempting the sync multiple times, I noticed that occasionally the import will appear to succeed for the first ~30 notes, then it fails for all subsequent notes. When I check OneNote after the conversion, the notebook is there, but it's empty.

Example attached.

[note-4e4b0bd5-757d-4927-be47-d4efbb827225.zip](https://github.com/stefankueng/EvImSync/files/12354089/note-4e4b0bd5-757d-4927-be47-d4efbb827225.zip)

**Comment by sdshaw** [comment]
Same issue here ... "occasionally the import will appear to succeed for the first ~30 notes, then it fails for all subsequent notes. When I check OneNote after the conversion, the notebook is there, but it's empty."

Question: the download states "Evernote2Onenote requires .NET Framework 4.8."  It does not say 4.8 "or later"_
I have more recent versions of .NET installed, 5.0.17 and 6.0.14 (both x64)
When I try to download and install 4.8 the install stops saying I have a more recent version of .NET installed already.
Could this be my/our problem? 

One other possibility, I used tags in Evernote.  Not every note, but enough that I estimate there are some tags in every notebook.  Could this be the reason why some notes appear to import, only to be followed by a series of individual failures?
This last attempt, Evernote2Onenote created two new (empty) notebooks, one was the same name as the notebook I was trying to import and one was clearly a tag.

**Comment by lanceruns** [comment]
SOLVED IT for myself.  

I also was having trouble getting through a full Evernote2Onenote conversion without errors.  I did two things to solve it: 
- on the export from Evernote, selected only the "date" check box, and unchecked all of the others
- before running Evernote2Onenote, I PAUSED ONEDRIVE syncing.  Then, after the import was done, I unpaused syncing and waited for the sync to finish before opening OneNote. 
With these changes, I was able to export and download large notebooks with over 500 notes in them. 

Good luck. 
Lance
